src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -397,18 +398,18 @@ protected function isCsrfTokenValid(string $id, ?string $token): bool
     /**
      * Dispatches a message to the bus.
      *
-     * @param object|Envelope $message The message or the message pre-wrapped in an envelope
-     *
+     * @param object|Envelope  $message The message or the message pre-wrapped in an envelope
+     * @param StampInterface[] $stamps
      * @final
      */
-    protected function dispatchMessage($message): Envelope
+    protected function dispatchMessage($message, array $stamps = []): Envelope
```

Comment:
this should be done in a BC way: the argument should be commented out and accessed using func_get_arg - see the code base for similar things.

Reply:
@nicolas-grekas thks for comment, changes made for BC way